sources/install.sh

```diff
@@ -2782,6 +2782,13 @@ function install_nmap() {
   add-test-command "nmap --version"
 }
 
+function install_netdiscover() {
+  colorecho "Installing netdiscover"
+  fapt netdiscover
+  add-history netdiscover
+  add-test-command "netdiscover -h"
```

Comment:
This test command `returns 1` and our checker check if the return is 0.
You can add this command : `add-test-command "netdiscover -h|& grep 'Usage: netdiscover'"`

Reply:
Alright, didn't knew how the testing was working ; thanks